registry/registry_test.go

```diff
@@ -282,6 +282,41 @@ func TestFactory_CreateCallRegistryWithLiveMetadata(t *testing.T) {
 	}
 }
 
+func TestFactory_CreateCallRegistry_Overrides(t *testing.T) {
+	var meta types.Metadata
+
+	err := codec.DecodeFromHex(test.CentrifugeMetadataHex, &meta)
+	assert.NoError(t, err)
+
+	t.Log("Metadata was decoded successfully")
+
+	// Lookup index for U64 in the test.CentrifugeMetadataHex
+	targetLookupIndex := int64(10)
```

Comment:
How does the user know about the target lookup index of the type to override?

Reply:
One would have to look in the metadata or registry when the metadata is parsed. Given the situation with missing/duplicated field names I think that this lookup index approach is the safest.